ibis/backends/base/sqlglot/datatypes.py

```diff
@@ -253,6 +253,8 @@ def _from_sqlglot_INTERVAL(
             else:
                 unit = precision_or_span.this.this
             return dt.Interval(unit=unit, nullable=nullable)
+        elif isinstance(precision_or_span, sge.Var):
+            return dt.Interval(unit=precision_or_span.this, nullable=nullable)
```

Comment:
This was caught by a hypothesis test failure.

Reply:
Do we have hypothesis tests for the sqlglot conversions?